crux-core/src/crux/api/alpha/CasOperation.java

```diff
@@ -5,6 +5,7 @@
 
```

Comment:
just in case I haven't mentioned it - intention is that `crux.api.alpha` gets deleted after your next block of work - if it's too much hassle to keep bringing it up to date, feel free to submit a PR deleting it

Reply:
I think they got hit when I was going through the code analysis auto-fixes. Was almost zero hassle :')
I think its best to leave in until the block of work is complete. Easier to refer to for any relevant inspiration